### PR TITLE
[wix-ui-tpa] Some package.json fixes

### PR DESCRIPTION
**Pull request by jonathanadler**
1. fix Janet deploy script (for releases)
2. fix lint commit hook to also add the files that were fixed to the commit

### CODE REVIEW

File: packages/wix-ui-tpa/package.json [hunk 1]
```diff
@@ -150,7 +150,8 @@
   },
   "lint-staged": {
     "*.{js,ts,tsx,scss,less}": [
-      "yoshi lint --fix"
+      "yoshi lint --fix",
+      "git add"
```
Comment (argshook) [inline packages/wix-ui-tpa/package.json:154]: idk, i wouldn't add `git add` cause it might accidentally add things
that were not meant to be committed

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6814

**Review by sivanharel — APPROVED** [review]

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6814/

File: packages/wix-ui-tpa/package.json (hunk 1 above)
Comment (jonathanadler) [reply packages/wix-ui-tpa/package.json]: you're right, I removed it

**Pull request merged**